.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
```

Comment:
I think we should stress that the overview should be non technical whenever possible. Ideally a PM should be able to read it and understand exactly what this PR does